src/file.cr

```diff
@@ -3,10 +3,9 @@ require "c/stdio"
 require "c/stdlib"
 require "c/sys/stat"
 require "c/unistd"
-require "path"
```

Comment:
I know what you mean. I'm not sure if you have seen my last comment on the PR. I am proposing to move some of the `File` methods which are currently just `Path.new(path).xyz.to_s` to path and remove the source methods and the `SEPARATOR*` constants from the source